src/pages/error-page/errorPage.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+
+import { useSelector , useDispatch } from 'react-redux';
+
+import { useStyles } from './errorPage.styles';
+import { getError } from '../../redux/error/error.actions';
+import { ERROR_PAGE_MESSAGE, ERROR_PAGE_IMAGES } from '../../configs';
+
+const ErrorPage = () => {
+  const { language, isLightTheme, errorMessage } = useSelector(
+    ({ Language, Theme, Error }) => ({
+      language: Language.language,
+      isLightTheme: Theme.lightMode,
+      errorMessage: Error.error
+    })
+  );
+
+  const classes = useStyles();
+  const dispatch = useDispatch();
+
+  const errorImagePath = isLightTheme
+    ? ERROR_PAGE_IMAGES.light
+    : ERROR_PAGE_IMAGES.dark;
+
+  useEffect(() => {
+    dispatch(getError());
```

Comment:
What does this action actually do?

Reply:
I changed the logic, it is no longer needed there